src/reducers/pause.js

```diff
@@ -73,53 +63,62 @@ function update(
     case constants.TOGGLE_PRETTY_PRINT:
       if (action.status == "done") {
         const frames = action.value.frames;
-        let pause = state.get("pause");
-        if (pause) {
-          pause = pause.set("frame", fromJS(frames[0]));
-        }
+        let pause = frames[0];
 
-        return state.merge({ pause, frames });
+        return Object.assign({}, state, { pause, frames });
       }
 
       break;
     case constants.BREAK_ON_NEXT:
-      return state.set("isWaitingOnBreak", true);
+      return Object.assign({}, state, { isWaitingOnBreak: true });
```

Comment:
I'd like to try keeping this an Immutable map, but just not hydrating the pause data.

I can jump in and look tonight